src/services/countries.js

```
@@ -0,0 +1,19 @@
+const axios = require('axios');
+
+const API_KEY = 'OWRxZlBwaXpIWXlYN3loZlB6cm1hdkhpWHVNeUthR1lFVlU3S21vWA==';
+const BASE_URL = 'https://api.countrystatecity.in/v1';
+
+const getCountries = async () => {
+  try {
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)